theano/tests/test_flake8.py

```diff
@@ -34,8 +34,8 @@
           'FI50', 'FI51', 'FI53')
 
 whitelist_flake8 = [
-    "compat/six.py",  # This is bundled code that will be deleted, don't fix it
     "__init__.py",
+    "_version.py",  # This is generated by versioneer
```

Comment:
I don't get it. This file is included in the repo, so I don't understand why it would be generated by versioneer

Reply:
verisoneer is a script that you run on your project and it generates a couple of files and installs them.  After that if you want to update you re-run the script with the new version and it will regenerate the files.

If we do local modifications, we will loose those modifications every update, which is a pain in the ass.